docker_files/recipes/python/meta/run.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: native
+config_set:
+  shell:
```

Comment:
With the proposal to include `$ARGS` this makes sense, yes. I'll include that in the new patch.